week-view.h

```diff
@@ -26,5 +26,6 @@ GtkWidget* week_view_new(void);
 void week_view_add_event(WeekView* cal, icalcomponent* vevent);
 void week_view_remove_event(WeekView* cal, icalcomponent* vevent);
 void week_view_add_calendar(WeekView* widget, Calendar* cal);
+int get_current_week(WeekView* wv);
```

Comment:
Could this please be `week_view_current_week` or `week_view_get_current_week`? Just to consistently follow glib/gtk naming conventions